src/modules/Classes.js

```diff
@@ -0,0 +1,45 @@
+export default class Books {
+  constructor(books) {
+    this.books = books;
+  }
+
+  get() {
+    return this.books;
+  }
+
+  // method to add book by geting data from dom
+  addBook(e) {
+    e.preventDefault();
+    const title = document.getElementById('title').value;
+    const author = document.getElementById('author').value;
+    this.books.push({ title, author });
+    localStorage.setItem('books', JSON.stringify(this.books));
```

Comment:
# Optional
- Please use an array method that doesn't mutate the array: Kindly use [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat) instead of [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push)

Reply:
@OLIPLICHE   Thank you for your review but I want to know the reason using concat methods. I mean if we use push it is very good for performance. The reason being that you said is because if we made many instances it will cause error and update all of them?. if it is I can just make set method and give empty array to `this.books`. Do you mean that way?

Reply:
Hi @heintayzar-hm, I am sorry for being late with this feedback. `Push()` is very good for performance, but it mutates the array. You are going to need the same "array" in the same array in the future, therefore `concat()` is needed.
Please Open this Link to learn more. [push() vs concat()](https://www.codingninjas.com/codestudio/library/difference-between-javascript-push-and-concat-methods)